apps/web/components/sections/about-us/problems.tsx

```
@@ -0,0 +1,48 @@
+
+  return (
+    <section className="py-16 bg-gray-100 flex flex-col items-center">
+      <div className="container mx-auto flex flex-col items-center">
+        <h2 className="text-3xl font-bold text-gray-900 text-center">The Problems We're Solving</h2>
+        <p className="text-gray-600 mt-2 max-w-2xl text-center">
+          Traditional crowdfunding faces significant challenges that we're addressing through innovative technology.
+        </p>
+        <div className="flex flex-wrap justify-center gap-4 mt-10">
+          {problems.map((problem, index) => (
+            <motion.div
+              key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)